custom_components/control4/binary_sensor.py

```diff
@@ -213,8 +213,9 @@ def __init__(
         )
         self._device_class = device_class
         self._extra_state_attributes["alarm_zone_id"] = alarm_zone_id
-        self._extra_state_attributes["ContactState"] = bool(
```

Comment:
What do these changes do? They seem unrelated to fan/climate support. Could you separate these changes out into a different PR?

Reply:
Yes, I have many more changes to the binary_sensor, I have found several different messages sent by different c4 binary sensors around the house.   I am not good with git, but I will try!